static/docs/tutorials/versioning.md

```diff
@@ -324,51 +324,52 @@ $ dvc run -f Dvcfile \
           python train.py
 ```
 
-Similar to `dvc add`, `dvc run` creates a DVC-file (forced to have file name
+Similar to `dvc add`, `dvc run` creates a DVC-file (forced to have the filename
 `Dvcfile` with the `-f` option). It puts all outputs (`-o`) under DVC control
-the same way as `dvc add` does. Unlike, `dvc add`, `dvc run` also tracks
+the same way that `dvc add` does. Unlike `dvc add`, `dvc run` also tracks
 dependencies (`-d`) and the command (`python train.py`) that was run to produce
-the result. We also such a DVC-file a "stage file".
+the result.
 
-> BTW, at this point you could `git add .` and `git commit` to save the
-> `Dvcfile` stage file and its changed output files to the repository.
+> At this point you could run `git add .` and `git commit` to save the `Dvcfile`
+> stage file and its changed output files to the repository.
 
-`dvc repro` will run `Dvcfile` if any of its dependencies (`-d`) changed, for
-example after we added new images like we did when we built the second model
-version. It also updates outputs and puts them into the <abbr>cache</abbr>.
+`dvc repro` will run `Dvcfile` if any of its dependencies (`-d`) changed. For
+example, after we added new images like we did when we built the second model
+version, that's a dependency change. It also updates outputs and puts them into
```

Comment:
Not sure we need to split this sentence. Maybe just shorten a little by removing "we did".

Reply:
I think it reads better this way. Feel free to change if you feel strongly

Reply:
You're right. I'm rewording a little anyway but keeping separate. Thanks